docs/azure/sdk/protocol-convenience-methods.md

```diff
@@ -0,0 +1,132 @@
+- Returns a friendly C# type that represents the result of the operation.
+
+---
+
+### Libraries that depend on System.ClientModel
+
+Some client libraries that connect to non-Azure services use patterns similar to the libraries that depend on `Azure.Core`. For example, the [`OpenAI`](https://www.nuget.org/packages/OpenAI/2.0.0-beta.7) library provides a client that connects to the OpenAI services. These libraries are based on a library called `System.ClientModel` that has patterns similar to `Azure.Core`.
+
+### [Protocol method](#tab/protocol-methods)
+
+The following code uses a `ChatClient` to call the `CompleteChat` protocol method:
+
+:::code source="snippets/protocol-convenience-methods/SCMProtocol/Program.cs" highlight="26-31":::
+
+The preceding code demonstrates the following `System.ClientModel` protocol method patterns:
+
+- Uses the `BinaryContent` type as a parameter to supply data for the request body.
+- Uses the `RequestContext` type to configure request options.
+- Returns data using the `ClientResult` type.
+- Calls the `GetRawResponse` method to access the response data.
+
+> [!NOTE]
+> The preceding code configures the `ClientErrorBehaviors.NoThrow` for the `RequestOptions`. This option prevents non-success service responses status codes from throwing an exception, which means the app code should manually handle the response status code checks.
+
+### [Convenience method](#tab/convenience-methods)
+
+Consider the following code that uses a `ChatClient` to call the `CompleteChat` convenience method:
+
+:::code source="snippets/protocol-convenience-methods/SCMConvenience/Program.cs" highlight="10,11":::
+
+The preceding code demonstrates the following `System.ClientModel` convenience method patterns:
+
+- Uses a standard C# primitive or model type as a parameter.
+- Returns a `ClientResult` type that represents the result of the operation.
+
+---
+
+## Protocol and convenience method usage guidance
+
+Although the Azure SDK for .NET client libraries provide the option to use either protocol or convenience methods, prioritize using convenience methods in most scenarios. Convenience methods are designed to improve the development experience and provide flexibility for authoring requests and handling responses. However, both method types can be used in your app as needed. Consider the following criteria when deciding which type of method to use:
+
+Convenience methods:
+
+- Enable you to work with more friendly method parameter and response types.
+- Handle various low-level concerns and optimizations for you.
+
+Protocol methods:
+
+- Provide access to lower-level types, such as `RequestContext` and `RequestOptions`, which aren't available through convenience methods.
+- Enable access to features of the underlying REST APIs that convenience methods don't expose.
+- Enable you to create your own convenience methods around service endpoints that don't already have convenience methods.
```

Comment:
Given the conversation around reading the REST API docs, it might be helpful to add to this list something along those lines -- maybe "Require understanding the service's REST API documentation to handle requests and responses correctly.